src/components/Menus/Sidebar/SettingsPanel/SettingsPanel.scss

```
@@ -0,0 +1,14 @@
+@use "../../../../spacing" as *;
+@use "../../../../font-size" as *;
+@import "../../../../colours.scss";
+
+.settings-panel {
+  color: $rpf-text-primary;
+  padding: $space-0-5;
+  margin-bottom: $space-0-5;
+
+  h3 {
+    font-weight: var(--font-weight-bold);
```

removing the default margin
```suggestion
    font-weight: var(--font-weight-bold);
    margin: 0 0 $space-0-5 0;
```